include/libgit4cpp/wrapper_functions.h

```diff
@@ -81,96 +82,94 @@ signature_default(git_repository* repo);
 
 /**
- * Create a new remote connection in the repository and returns it
- * \param repo C-type repository
- * \param remote_name typically 'origin'
- * \param url adress of remote connection, e.g https://github.com/...
- * \return Wrapper of a remote connection
+ * Create a new remote connection in the repository.
+ * \param repo Pointer to repository object
+ * \param remote_name Name of the remote, e.g. "origin"
+ * \param url Adress of remote connection, e.g https://github.com/...
+ * \return new git_remote object
  */
 LibGitPointer<git_remote>
 remote_create (git_repository* repo, const std::string& remote_name,
                 const std::string& url);
 
 
 /**
- * Collects the remove connection by name and returns it
- * \param repo C-type repository
- * \param remote_name typically 'origin'
- * \return Wrapper of a remote connection
+ * Collects the remove connection by name.
+ * \param repo Pointer to repository object
+ * \param remote_name Name of the remote, e.g. "origin"
+ * \return new git_remote object
  */
 LibGitPointer<git_remote>
 remote_lookup (git_repository* repo, const std::string& remote_name);
 
 
 
 /**
- * Clone existing git repository into local filesystem
- * \param url adress of remote connection, e.g https://github.com/...
- * \param repo_path absolute or relative path from executable
- * \return Wrapper of a repository
+ * Clone existing git repository into local filesystem.
+ * \param url Address of remote connection, e.g https://github.com/...
+ * \param repo_path Absolute or relative path to the repository root
+ * \return new git_repository object
 */
 LibGitPointer<git_repository>
 clone (const std::string& url, const std::string& repo_path);
 
 
 /**
- * Find the reference to a branch where the name is known
- * \param repo C-type repository
- * \param branch_name e.g. 'master', '3_fix bugs', etc...
- * \param branch_type enum with 1=GIT_BRANCH_LOCAL, 2=GIT_BRANCH_REMOTE, 3=GIT_BRANCH_ALL
- * \return Wrapper of git reference
+ * Find a named branch.
+ * \param repo Pointer to repository object
+ * \param branch_name The name to search for, e.g. 'main', 'fix-bugs'
+ * \param branch_type Which branch type to find, enum with 1=GIT_BRANCH_LOCAL, 2=GIT_BRANCH_REMOTE, 3=GIT_BRANCH_ALL
+ * \return new git_reference object
 */
 LibGitPointer<git_reference>
 branch_lookup(git_repository* repo, const std::string& branch_name, git_branch_t branch_type);
 
 
 /**
- * Return the name of a remote name
- * \param repo C-type repository
- * \param branch_name e.g. 'master', '3_fix bugs', etc...
- * \return remote name, typically 'origin/master' or 'origin/3_fix_bugs'
- * 
+ * Find the name of a branch on the remote.
+ * \param repo Pointer to repository object
+ * \param branch_name Local branch name, e.g. 'main', 'fix-bugs'
+ * \return name on remote , e.g. 'origin/main' or 'origin/fix-bugs'
 */
 std::string
 branch_remote_name(git_repository* repo, const std::string& branch_name);
 
-/** \}*/
```

Comment:
I believe this comment closes the group opened above with `\defgroup`?

Reply:
Ah yes, never heard about such groups.

Reintroduced the closing curly, but - as suggested by the Doxygen docs - put an additional comment there that this closes the group:

<img width="925" alt="Screenshot 2023-12-06 at 17 51 07" src="https://github.com/taskolib/libgit4cpp/assets/16012374/adf28a84-26ef-42bb-a709-92962b1526fd">